### PR TITLE
tools/rbd: Format the output text in do_show_info()

### PR DESCRIPTION
**Pull request by chuanhong-wang**
check and edit the output of  command "rbd info" and "ceph -s"

### CODE REVIEW

**Comment by joscollin** [comment]
Jenkins Retest this please

File: src/osd/OSDMap.cc [hunk 1]
```diff
@@ -3195,7 +3195,7 @@ void OSDMap::print_summary(Formatter *f, ostream& out) const
     f->dump_unsigned("num_remapped_pgs", get_num_pg_temp());
     f->close_section();
   } else {
-    out << get_num_osds() << " osds: "
+    out << get_num_osds() << " osds, "
```
Comment (tchaikov) [inline src/osd/OSDMap.cc]: iirc, @liewegas also have a change for this.

**Comment by dillaman** [comment]
@chuanhong-wang Please add the "rbd: " prefix to your commit subject line

**Comment by joscollin** [comment]
Jenkins Retest this please

**Comment by dillaman** [comment]
@chuanhong-wang ping

**Comment by chuanhong-wang** [comment]
@dillaman , Sorry for reponse so late. Thanks @joscollin for help me added "rbd:" in pr title. what else need I do? I have done "make check" for this pr on my servers, sometimes it passed, and sometimes failed, I didn't find the reason. Maybe I should cancel this pr if it can't pass all the test?

**Comment by dillaman** [comment]
retest this please

**Comment by dillaman** [comment]
cli test case needs to be updated: http://qa-proxy.ceph.com/teuthology/jdillaman-2017-08-17_14:06:40-rbd-wip-jd-testing-distro-basic-smithi/1536056/teuthology.log

**Comment by dillaman** [comment]
@chuanhong-wang ping

**Comment by dillaman** [comment]
@chuanhong-wang ping

**Comment by dillaman** [comment]
@chuanhong-wang Please feel free to re-open if you can fix the CLI test case.

**Pull request closed without merging**